src/hotspot/share/runtime/crac.cpp

```diff
@@ -289,6 +289,10 @@ static bool ensure_checkpoint_dir(const char *path, bool rm) {
   return true;
 }
 
+#ifndef PATH_MAX
+# define PATH_MAX 1024
+#endif
```

Comment:
Not critical, but we could also try `MAX_PATH` defined in Windows before falling back to a hard-coded value